### PR TITLE
Update buttons

### PR DESCRIPTION
**Pull request by dcmouyard**
This pull request depends on #46 being merged first.

### CODE REVIEW

**Comment by dcmouyard** [comment]
I updated the default button styles to match the [U.S. Web Design Standards](https://standards.usa.gov/colors/).

This pull request depends on #51 being merged first.

**Comment by dcmouyard** [comment]
This pull request now depends on #46, #51, and #52 being merged first.

File: sass/partials/components/_button.scss [hunk 1]
```diff
@@ -3,15 +3,62 @@
...
       [dir='rtl'] & {
         margin-left: 0;
-        margin-right: 1em;
+        margin-right: 0.5em;
       }
     }
   }
 }
+
+.button--danger {
```
Comment (coreylafferty) [inline sass/partials/components/_button.scss]: Do we need all of these extra button styles?  Are they just examples or are any of these classes actually output in Drupal?

**Review by coreylafferty — CHANGES_REQUESTED** [review]

File: sass/partials/components/_button.scss (hunk 1 above)
Comment (dcmouyard) [reply sass/partials/components/_button.scss]: The .button and .button--danger classes are the only ones that Drupal 8 outputs by default. The others are just examples based on what I’ve seen on projects the most. The designs for these button all come from https://standards.usa.gov/buttons/

Comment (coreylafferty) [reply sass/partials/components/_button.scss]: Can you add comments for the variants to that effect?  e.g., 

// Default Drupal button class output during the zombie apocalypse.
.button--danger {}

// Custom button class, included as an example. Not a default Drupal class name.
.button--secondary {}

// Size variations, included as examples.  Not default Drupal class names.
.button--lg {}
.button--sm {}

**Review by coreylafferty — CHANGES_REQUESTED** [review]

**Comment by dcmouyard** [comment]
I broke out the button variants to a sub-folder and added comments explaining what they’re for.

**Review by coreylafferty — APPROVED** [review]

**Pull request merged**